concent_api/conductor/tests/test_integration_verification.py

```diff
@@ -0,0 +1,210 @@
+import mock
+
+from django.test    import override_settings
+from django.test    import TestCase
+from django.urls    import reverse
+
+from golem_messages import message
+
+from ..models       import BlenderSubtaskDefinition
+from ..models       import UploadReport
+from ..models       import VerificationRequest
+from ..tasks        import blender_verification_order
```

Comment:
That function exist ? `ImportError: cannot import name 'blender_verification_order'` The same in `views.py`.